docs/testing.md

```diff
@@ -284,4 +284,21 @@ S3_FOLDER=snapshots
 ### Not supported/implemented currently for cluster restore:
 - Hardened Cluster Setup
 - ExternalDB Setup
-- Selinux Setup
\ No newline at end of file
+- Selinux Setup
+
+## Validating Conformance Tests with Sonobuoy
+ - Please note that the sonobuoy version has not been updated for a year and the functionality of sonobuoy is degrading with minor versions of k8s.
+ - Full conformance tests done for patch validations should be 3 servers 1 agent minimum.
+ - You can use the make file command `make test-run-sonobuoy` to run the conformance tests.
+ - Additionally you can use `go test -timeout=140m -v -count=1 ./entrypoint/conformance/... --ginkgo.timeout=140m` you must extend the ginkgo timeout in addition to the go test timeout 
```

Comment:
is this the minimum timeout? does timeout vary based on the product?

Reply:
This timeout was for RKE2, I did notice Fran expanded it even larger perhaps to accommodate running through Jenkins.